cmd/utils/flags.go

```diff
@@ -228,7 +228,7 @@ var (
 		Usage: `Blockchain garbage collection mode ("full", "archive")`,
 		Value: "full",
 	}
-	SnapshotFlag = cli.BoolTFlag{
+	SnapshotFlag = cli.BoolFlag{
```

Comment:
What's the default value of `BoolFlag`? in V1, `BoolTFlag` has `True` as the default value.

Reply:
In that case, all changes from `BoolTFlag` to `BoolFlag` must also set the `Value:  true,` as specified in those docs.

Reply:
Just updated `cmd/utils/flags.go`, will keep looking for more occurrences.